boards/st/nucleo_u5a5zj_q/nucleo_u5a5zj_q-common.dtsi

```diff
@@ -55,22 +55,30 @@
 	status = "okay";
 };
 
+/* This board has a 16MHz crystal attached */
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(16)>;
+	status = "okay";
+};
+
 &clk_lse {
 	status = "okay";
 };
 
+/* Note: this is configured as 4MHz source clock */
 &clk_msis {
 	status = "okay";
 	msi-range = <4>;
 	msi-pll-mode;
 };
 
 &pll1 {
-	div-m = <1>;
-	mul-n = <80>;
-	div-q = <2>;
-	div-r = <2>;
-	clocks = <&clk_msis>;
+	/* HSE 16MHz source, outputting 160MHz to sysclk and apbclk */
+	div-m = <4>; /* input divisor */
+	mul-n = <80>; /* VCO multiplication factor */
```

Comment:
Any specific reason why to use 4 and 80 instead of 1 (div-m) and 20 (mul-n)?

Reply:
I just wanted to keep things as close to the original divisors in case of other peripherals/weirdness.

Reply:
Okay, makes sense.